contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -99,7 +99,7 @@ contract KyberFeeHandler is IKyberFeeHandler, DaoOperator, Utils5 {
     event BurnConfigSet(ISanityRate sanityRate, uint256 weiToBurn);
     event RewardsRemovedToBurn(uint256 indexed epoch, uint256 rewardsWei);
     event KyberNetworkUpdated(address kyberNetwork);
-    event KyberProxyUpdated(IKyberProxy indexed newProxy, IKyberProxy indexed oldProxy);
+    event KyberProxyUpdated(IKyberProxy indexed kyberProxy);
```

Comment:
seems unclear why proxy address is indexed and network not.
I think index not required in both events
but if it is required.
please index in both.

Reply:
Ok removed both, seems to me updating network or proxy addresses is rare